apps/frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -49,21 +49,19 @@ const Sidebar: React.FC<Properties> = ({
 
 			<div className={styles["logo-container"]}>Logo</div>
 			<div className={styles["buttons-container"]}>
-				<div className={styles["navlinks-container"]}>
```

Comment:
Did you remove styles as well? If they are not used anymore

Reply:
yes

Reply:
yes, it's in commit [556545b](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/pull/329/commits/556545ba777e4092b4c8deef13b67800f2f9264b)